continuous_integration/recipe/conda_build_config.yaml

```
@@ -4,5 +4,10 @@ python:
     - 3.10
 c_compiler_version:
     - 11
+libprotobuf:
+    - 3
 rust_compiler_version:
     - 1.69
```

```suggestion
setuptools_rust:
    - 1.5.2
```

Or maybe some newer version if desired